src/applications/accredited-representative-portal/accreditation/21a/app-entry.jsx

```diff
@@ -1,31 +1,5 @@
-import '@department-of-veterans-affairs/platform-polyfills';
-
 import React from 'react';
-import { Provider } from 'react-redux';
-import { Router, useRouterHistory } from 'react-router';
-import { createHistory } from 'history';
-
 import startReactApp from '@department-of-veterans-affairs/platform-startup/react';
-import { connectFeatureToggle } from 'platform/utilities/feature-toggles';
-
-import './sass/21a.scss';
-
 import manifest from './manifest.json';
-import routes from './routes';
-import createReduxStore from '../../store';
-import rootReducer from './reducers';
-
-window.appName = manifest.entryName;
-const store = createReduxStore(rootReducer);
-connectFeatureToggle(store.dispatch);
-
-// eslint-disable-next-line react-hooks/rules-of-hooks
-const history = useRouterHistory(createHistory)({
-  basename: manifest.rootUrl,
-});
 
-startReactApp(
-  <Provider store={store}>
-    <Router history={history} routes={routes} />
-  </Provider>,
-);
```

Comment:
This is just for the install. The 21a app isn't part of the pilot so simplifying the entry file for 21a for the upgrade install. We'll have to work out a replacement for the history usage to reenable.